browser/net/brave_site_hacks_network_delegate_helper_browsertest.cc

```diff
@@ -296,11 +291,11 @@ IN_PROC_BROWSER_TEST_F(BraveSiteHacksNetworkDelegateBrowserTest,
 
 IN_PROC_BROWSER_TEST_F(BraveSiteHacksNetworkDelegateBrowserTest,
                        QueryStringFilterShieldsDown) {
-  const std::string inputs[] = {
+  static constexpr std::string_view kInputs[] = {
       "", "foo=bar", "fbclid=1", "fbclid=2&key=value", "key=value&fbclid=3",
   };
 
-  for (const auto& input : inputs) {
+  for (const auto& input : kInputs) {
```

Comment:
Are there any strict rules whether we should capture string_view either by ref or by value? Or both are equivalent? 
Just comparing to span which is typically always copied.